polaris-react/src/components/CheckableButton/CheckableButton.scss

```diff
@@ -86,8 +86,8 @@ $button-vertical-padding: calc(
   height: var(--p-choice-size);
   width: var(--p-choice-size);
   margin-left: calc(
-    -1 * ((var(--p-space-2) + var(--p-space-025))) - var(--p-control-border-width)
-  ); // 1px accounts for border
+    -1 * (var(--p-space-2) + var(--p-space-025))
```

Comment:
Oof. I wish this used a token.

Reply:
This would probably be a one off token

Reply:
I don't mean we should add it now. This should be refactored in the future to not do this. Ship though, this alignment is important.